org.mixedrealitytoolkit.uxcore/Button/Experimental/ExperimentalPressableButtonWithDynamicFrontPlate.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace MixedReality.Toolkit.UX
+{
+    /// <summary>
+    /// This mono behaviour is used to enable or disable the front plate of a button when the button enters or exists proximity-hovering.
+    /// </summary>
+    [RequireComponent(typeof(PressableButton))]
+    public class ExperimentalPressableButtonWithDynamicFrontPlate : MonoBehaviour
```

Comment:
I don't understand the need for this class.   You've created a base "Empty Button" template.  You 
can hook up then event in the "Empty Button (Experimental)"'s inspector.  No need for code.

Reply:
I was trying to keep all the Experimental code as separated as possible from the release code so that it would be easy to identify and update as needed.  Also, I added the class so that existing buttons wouldn't have their behavior modified by this experimental code.

Reply:
Implemented with event hooking directly in Inspector in latest commit.  Resolving this.